### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#7171] [Q&A] [answered]
Hello everyone!

I want to help, but I want to know if this project is active. *Is this project active?*

- I would like to know how I can contribute to chakra-ui.

**Comment by ArthurPedroti** [comment, marked as answer]
Yes my friend, very active, help us to improve even more🚀

**Reply by ghost** [reply]
Hi ArthurPedroti.

> Yes my friend, very active, help us to improve even morerocket

thanks! thank you for feedback!!!!
